docker/scripts/entrypoint.sh

```diff
@@ -1,6 +1,16 @@
 #!/usr/bin/env bash
 set -eEuo pipefail
 
+get_arg_name_from_env_name() {
```

Comment:
I would document a bit this function, or better, I would add a few comments at the beginning of the file explaining which is the syntax of the env variables and how the get converted into `nh-node` parameters, also citing how setting a NH_CONF_XYZ env variable to `"true"` is treated differently (i.e. converted to a argument-less command line option).

Reply:
Your suggestion is good, I have added documentation on top of the script.